incubation_process.md

```diff
@@ -41,20 +42,22 @@ The barrier for entering the Sandbox stage is intended to be low, so that it's e
 
 ## Stage 2: Incubation
 
-The Incubation stage is for projects that have reached a level of maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+The Incubation stage is for projects that have reached a high level of quality and maturity that indicates they are ready to be used by end users. Incubation projects meet the main criteria of the OpenRail Association which make sure that the project is governed openly. Open governance means that decisions are taken in a transparent way and there is a path to become committer and maintainer based on the merits of contributions.
+
+The Incubation stage focuses on quality criteria the project can reach on its own and doesn't require a high degree of external adoption or contributions yet.
```

Comment:
What about asking for a public version like NGE did. I think from the business point of view, it could be very usefull.

Reply:
Even though this can be useful for software with a UI for the business user, I do not see it as a requirement since it hardly makes sense for libraries, for example.

Reply:
I would see it as Peter. It's certainly a good practice and can be beneficial for the project and people who want to explore it. But requiring it as part of the incubation criteria would be too detailed for me.